zipline/data/data_portal.py

```diff
@@ -841,11 +901,17 @@ def _get_minute_spot_value(self, asset, column, dt, ffill=False):
         if dt == last_traded_dt or dt.date() == last_traded_dt.date():
             return result
 
+        # If we are in before_trading_start, dt is 4pm the previous day, and
```

Comment:
See comment above.  This seems unlikely to be correct on the days after non-trading days.